pallets/handles/src/handles-utils/build.rs

```diff
@@ -14,6 +14,10 @@ fn allowed_char(c: char) -> bool {
 
 #[allow(clippy::unwrap_used)]
 fn main() {
+	println!("cargo:rerun-if-changed=build.rs");
+	println!("cargo:rerun-if-changed=src/data/confusable_characters.txt");
+	println!("cargo:rerun-if-changed=constants.rs");
+
```

Comment:
This should be at the top and `src/data` not `data` was causing it to run everytime.